cache.go

```diff
@@ -135,6 +135,17 @@ func (c *cache) Get(k string) (interface{}, bool) {
 	return item.Object, true
 }
 
+//GetMultipleItems returns an array of items corresponding to the input array
+func (c *cache) GetMultipleItems(values []string) []interface{} {
```

Comment:
should it really be called `GetMultipleItems`, it is a bit stuttery to me, look that the Get function is not called GetItem